docs/Hosts/Spawn-Hosts.md

```diff
@@ -55,6 +55,149 @@ lifetime up to 30 days past host creation.
 If you'd like to get a notification before a host expires, you can [set up a
 notification](../Project-Configuration/Notifications#spawn-host-expiration) for it.
 
+### Unexpirable Host Sleep Schedules
+
+**Summary: Please set a sleep schedule for your unexpirable hosts by editing them [in the
+UI](https://spruce.mongodb.com/spawn/host) before `<DATE>`. In addition, please opt into the beta test to ensure that
+your sleep schedule is working the way you want.**
+
+For unexpirable hosts, Evergreen has introduced a new feature called a host sleep schedule, which allows you to control
+when you'd like your unexpirable host to be automatically turned on or off. A sleep schedule lets you choose recurring
+times of the week when you want your host to be on, and when it's okay for it to be turned off. For example, you could
+set a sleep schedule where your host is on during your work hours from 10 am to 6 pm between Monday and Friday and
+otherwise let it turn it off overnight and during weekends. Setting a sleep schedule ensures your host is on when you're
+working, while also stopping the host when it's not being actively used. This is important to ensure that hosts are
+being efficiently utilized and to avoid over-spending on long-lived AWS resources when they're idle.
+
+Please note that even though Evergreen will automatically stop and start your host at the configured times, you can
+still manually stop and start your host whenever you need. It's recommended that if you do this, you also [set a
+temporary exemption](#temporary-exemptions) to ensure that you can use your host without being interrupted when the
```

Comment:
I think it might not be immediately obvious why a temporary exemption is needed in conjunction with starting or stopping a host manually. Can we maybe elaborate on top of "to ensure that you can use your host without being interrupted when the sleep schedule tries to automatically stop/start your host"? Maybe something like "so that that the sleep schedule doesn't resume before you're ready for it, for example if you'd like for it to stay off until you're back from PTO, or stay on for a day so that a test can finish running"?

Reply:
Those examples + the whay are already explained in detail in the temporary exemption section. This is just a note that you should be aware that the sleep schedule will still happen regardless of manual stops/starts (and to go there to read more on the why).